app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -12,43 +14,73 @@
 @Service
 @Transactional
 public class ProductService {
+    private final Mapper mapper;
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(
+            Mapper dozerMapper,
+            ProductRepository productRepository) {
+        this.mapper = dozerMapper;
         this.productRepository = productRepository;
     }
 
+    /**
+     * 상품 목록을 리턴한다.
+     *
+     * @return 상품목록
+     */
+
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * id에 해당하는 상품을 리턴한다
```

Comment:
다른 JavaDoc과 통일성있게 점을 찍어주도록 합시다.
```suggestion
     * id에 해당하는 상품을 리턴한다.
```

Reply:
넵 전체적으로 수정하겠습니다